README.md

```diff
@@ -34,54 +43,48 @@ An example of the provenance generated from this repo is below:
     {
       "name": "binary-linux-amd64",
       "digest": {
-        "sha256": "0ae7e4fa71686538440012ee36a2634dbaa19df2dd16a466f52411fb348bbc4e"
+        "sha256": "7bf2e6ebb97e1bdb669d9df73048247f141e2f8e72ab59f23d456f1bc5a041dc"
       }
     }
   ],
   "predicate": {
     "builder": {
-      "id": "https://github.com/slsa-framework/slsa-github-generator-go/.github/workflows/builder.yml@main"
+      "id": "https://github.com/ianlewis/slsa-github-generator-go/.github/workflows/builder.yml@refs/heads/library"
     },
     "buildType": "https://github.com/slsa-framework/slsa-github-generator-go@v1",
     "invocation": {
       "configSource": {
-        "uri": "git+https://github.com/asraa/slsa-on-github-test@refs/heads/main.git",
+        "uri": "git+https://github.com/ianlewis/actions-test@refs/heads/main",
         "digest": {
-          "sha1": "11dba28bf106e98f9992daa56e3967be41a5f11d"
+          "sha1": "d29d1701b47bbbe489e94b053611e5a7bf6d9414"
         },
-        "entryPoint": "Test SLSA"
-      },
-      "parameters": {
-        "version": 1,
-        "event_name": "workflow_dispatch",
-        "ref_type": "branch",
-        "ref": "refs/heads/main",
-        "base_ref": "",
-        "head_ref": "",
-        "actor": "asraa",
-        "sha1": "11dba28bf106e98f9992daa56e3967be41a5f11d",
-        "event_payload": ...
+        "entryPoint": "build w/ SLSA provenance"
```

Comment:
want to update this to the filepath? Or wait until [it's implemented (I recall you said we can get it with `github.input` variable)](https://github.com/slsa-framework/slsa-github-generator/issues/11) is landed?

Reply:
Yeah, I think eventually we'll do that as part of https://github.com/slsa-framework/slsa-github-generator/issues/11